artwork_redirect/request.py

```diff
@@ -185,7 +220,37 @@ def resolve_cover(self, mbid, type, thumbnail):
         raise NotFound("No %s cover image found for release with identifier %s" % (
             type.lower(), mbid))
 
-    def resolve_image_id(self, mbid, filename, thumbnail):
+    def resolve_event_art(self, mbid, type, thumbnail):
+        """Get the frontiest artwork image."""
+
+        if type == "Front":
```

Comment:
Yeah, it's just the "most representative" image for the event, but the terminology was kept the same for simplicity (I think the IA had asked about it, since they use the "front" flag to determine what image to show on their details page).

Reply:
Oh, ok :)